### DISCUSSION TITLE
Discussion | Tuesday, 3 December 2024, 8:00 PM

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#72] [Zoom Links]
Caawa aan isku imaano Insha'Allah!

Zoom Link:
https://us06web.zoom.us/j/81103000545?pwd=DkzLPVlZbAsEJt7C9aPVO3W1fzTZdb.1
Mentions

@dugsiiyeinc/dhaqsopay-extension-developers
@dugsiiyeinc/dhaqsopay-server-developers
@dugsiiyeinc/dhaqsopay-website-developers

**Comment by shiinedev** [comment]
Ok I.a

**Comment by iftiiin** [comment]
Ok eng

**Comment by Nasrah-muse** [comment]
Okay eng diyaar

**Comment by abdinasir-Tman** [comment]
it's time @dugsiiyeinc/dhaqsopay-contributors waan joogaa

**Comment by hayle01** [comment]
Ready

**Comment by A-M-Ahmed** [comment]
Cool wllka
